app/controllers/application_controller.rb

```diff
@@ -5,6 +5,67 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions unless test?
+    set :session_secret, "secret"
   end
 
+  get '/' do
+  erb :index
+  end
+
+  get '/tweets/new' do
+    erb :'tweets/create_tweet'
+  end
+
+  post '/tweets' do
+    tweet = Tweet.create(params)
```

Comment:
the `create()` method already commits to the database so `.save` accomplishes nothing here for us.